examples/dotnet_core/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine AS build
+LABEL maintainer=""
+
+WORKDIR /app
+COPY . .
+RUN dotnet restore \
+    --runtime linux-x64 \
+    && dotnet publish \
+    --configuration Release \
+    --output /app/out \
+    --runtime linux-x64
+
+
+# FROM gcr.io/distroless/dotnet_core_runtime:dotnet_core_runtime_debian10 as runtime
+# FROM gcr.io/distroless/dotnet/core:3.1 as runtime # Aim for this container image tag.
+
+# Use below local images for local development.
+FROM bazel/experimental/dotnet_core_runtime:dotnet_core_runtime_debian10 as runtime
+# FROM bazel/experimental/dotnet_core_runtime:debug_debian10 as runtime
+LABEL maintainer=""
+
```

Comment:
```suggestion
# FROM gcr.io/distroless/dotnet_core_runtime:dotnet_core_runtime_debian10 as runtime
FROM gcr.io/distroless/dotnet/core:3.1 as runtime # Aim for this container image tag.
```

Mentioning `bazel/experimental` will be confusing to regular Distroless consumers (i.e., users who don't develop Distroless).

Reply:
```Dockerfile
# FROM gcr.io/distroless/dotnet_core_runtime:dotnet_core_runtime_debian10 as runtime
FROM gcr.io/distroless/dotnet/core:3.1 as runtime # Aim for this container image tag.
```

I cannot use these FROM lines because it is a non-existent Docker container image at this stage?

Reply:
The images will be auto-published when this PR is merged. That said, I think the images names are wrong. They should be

```Dockerfile
# FROM gcr.io/distroless/dotnet/core/runtime as runtime
FROM gcr.io/distroless/dotnet/core/sdk:3.1 as runtime # Aim for this container image tag.
LABEL maintainer=""
```

Reply:
Thank for the good catch. PR updated.